Gemfile

```diff
@@ -13,7 +13,7 @@ group :development do
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'rake', '10.4.2'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.43.0'
```

Comment:
Actually, I think this is technically not required. If you empty your gems and remove the Gemfile.lock all is good. You will automatically get 0.43. It is a bit sneaky to add this here.